action.yml

```diff
@@ -16,6 +17,22 @@ inputs:
     description: 'Limit the number of first x lines from commit messages that will be added in the issue description. No truncation when set to 0.'
     required: false
     default: 1
+
 runs:
-  using: 'node16'
-  main: 'dist/index.js'
+  using: 'composite'
+  steps:
+    - name: Checkout the fresh project
+      uses: actions/checkout@v3
+
+    - name: Fetch the tags
+      run: |
+        git fetch
+        git fetch --tags
+      shell: bash
+
+    - name: Run the action javascript
+      uses: actions/github-script@v6
+      with:
+        notify-after: ${{ inputs.notify-after }}
```

Comment:
this is not an input that we need to provide to the action. inputs are not being provided to the script below, which is clearly a problem and the action, as it is now, cannot work. we had already discussed this if you remember, and I linked the example of what we're doing in another action by using something along the lines of https://github.com/nearform/optic-release-automation-action/blob/main/action.yml#L100

Reply:
the problem is, we have more inputs! so either we pass all them in this way, or, more reasonably, we use the approach I recommended

Reply:
running the action I get this warning:

<img width="1385" alt="image" src="https://user-images.githubusercontent.com/6604111/215968310-54e05dfb-e39b-4b41-b5f0-3d8053bf6cb7.png">

this could be related to the highlighted piece of code